templates/default/systemd.service

```diff
@@ -16,6 +16,8 @@ Type = forking
 Environment="<%= key %>=<%= value %>"
 <% end -%>
 <% end -%>
+Restart=always
+TimeoutSec=420
```

Comment:
Remove the TimeoutSec from here. The correct place to change it is from the install recipe (which you did).